applications/surgical_scene_recon/README.md

```
@@ -0,0 +1,428 @@
+./holohub run surgical_scene_recon dynamic_rendering
+```
+
+Checkpoint location: `applications/surgical_scene_recon/output/trained_model/ckpts/fine_best_psnr.pt`
+
+---
+
+## Advanced Usage
+
+### Troubleshooting
+
+**Problem: "FileNotFoundError: poses_bounds.npy not found"**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)